lib/FilesHooks.php

```diff
@@ -282,6 +283,7 @@ public function fileMove($oldPath, $newPath) {
 			$oldAccessList['users'] = array_merge($oldAccessList['users'], $this->getAffectedUsersFromCachedMounts($oldFileId));
 		}
 
+		$this->moveCase = $moveCase;
```

Comment:
Could you add a comment here? I am not sure to understand why you are caching the value before assigning it?